webapp/lib/charts.dart

```diff
@@ -142,17 +145,20 @@ class DailyTimeseriesLineChartView {
     chart = chartjs.Chart(canvas.getContext('2d'), chartConfig);
   }
 
-  void updateChart(Map<DateTime, int> updatedCountsPerDay) {
-    List<chartjs.ChartPoint> timeseriesPoints = [];
-    List<DateTime> sortedDateTimes = updatedCountsPerDay.keys.toList()
-      ..sort((t1, t2) => t1.compareTo(t2));
-    for (var datetime in sortedDateTimes) {
-      var value = updatedCountsPerDay[datetime];
-      timeseriesPoints.add(new chartjs.ChartPoint(t: datetime.toIso8601String(), y: value));
-    }
-    chartData.datasets.first.data
+  void updateChart(List<Map<DateTime, int>> updatedCountsAtTimestampList) {
+    for (var i =0; i<updatedCountsAtTimestampList.length; i++) {
+      List<chartjs.ChartPoint> timeseriesPoints = [];
+      List<DateTime> sortedDateTimes = updatedCountsAtTimestampList[i].keys.toList()
+        ..sort((t1, t2) => t1.compareTo(t2));
+      for (var datetime in sortedDateTimes) {
+        var value = updatedCountsAtTimestampList[i][datetime];
+        timeseriesPoints.add(
+            new chartjs.ChartPoint(t: datetime.toIso8601String(), y: value));
+      }
+      chartData.datasets[i].data
       ..clear()
       ..addAll(timeseriesPoints);
```

Comment:
nit: these should be indented by 2 spaces

Reply:
Added the indent